src/utils.js

```diff
@@ -52,39 +82,56 @@ function getComponentNames(basePaths) {
     return names.sort();
 }
 
-/**
- * @param gmeConfig
- * @param logger
- * @param callback
- * @returns {*}
- */
-function copySvgDirsAndRegenerateSVGList(gmeConfig, logger, callback) {
-    var deferred = Q.defer(),
-        svgAssetDir = path.join(__dirname, 'client', 'assets', 'DecoratorSVG');
+function getSVGMap(gmeConfig, logger, callback) {
```

Comment:
couldn't it be even synchronous?
I know it would be blocking, but we do not expect too deep and too large sets of files (so it should be fairly fast).